lib/profiling/Makefile

```diff
@@ -45,6 +45,9 @@ F90FLAGS ?= -g
 # since they require external libraries to be available.
 ALL_LIBS = lfric_timer simple_timing template
 
+# For convenience, define a list with all libraries (which will a)
```

Comment:
"which will a"? :-)

Reply:
I have no idea how that sentence was supposed to end. But I did a completely rename, and update of many comments there, it's much better now, and hopefully all sentences now have a proper ending :)